dev/tasks/dev.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * @license Copyright (c) 2003-2015, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+const initTask = require( './utils/dev-init' );
+const pluginCreateTask = require( './utils/dev-plugin-create' );
+const pluginInstallTask = require( './utils/dev-plugin-install' );
+const pluginUpdateTask = require( './utils/dev-update' );
+const pluginStatusTask = require( './utils/dev-status' );
+const boilerplateUpdateTask = require( './utils/dev-boilerplate-update' );
+const ckeditor5Path = process.cwd();
+
+module.exports = ( grunt ) => {
+	const packageJSON = grunt.config.data.pkg;
+
+	grunt.registerTask( 'dev-init', function() {
+		const options = getOptions( this );
+		initTask( ckeditor5Path, packageJSON, options, grunt.log.writeln, grunt.log.error );
+	} );
+
+	grunt.registerTask( 'dev-plugin-create', function() {
+		const done = this.async();
+		const options = getOptions( this );
+		pluginCreateTask( ckeditor5Path, options, grunt.log.writeln, grunt.log.error ).then( done );
```

Comment:
I know that it is (or will be) handled inside `pluginCreateTask()` but if this function returns a promise it's necessary to handle it. Otherwise, if after some refack code gets broken somewhere "outside" that handler which is used inside `pluginCreateTask()` then you won't know what's going on.